draft-ietf-quic-tls.md

```diff
@@ -1133,27 +1133,6 @@ sets available to a server one-round trip earlier than parameters that are
 carried in QUIC packets.  This document does not define that extension.
 
 
-## Source Address Validation {#source-address}
```

Comment:
So I admit that I don't understand what @janaiyengar has in mind, but if you wanted to use these mechanisms, you could.  If you ALSO wanted to keep the logical split, then you say "the crypto handshake is responsible for performing source address validation" and then we shuffle things around.

I didn't recognize that option when I originally put this together.  See the mailing list thread for more.